portworx/charts-rancher
# PWX-24857, PWX-24785 upgrade px-essentials to 2.10.3 and add px-role with secrets permissions

**What this PR does / why we need it**:
Update px-essential to 2.10.3 and stork to 2.9.0 as specified in the install daemon. Bonus: adds px-role secrets

**Which issue(s) this PR fixes** (optional)
PWX-24857, PWX-24785

**Special notes for your reviewer**:
Tested on local cluster

Commits should be in the order below, it's easier to review just commit 2 -4
1. Copy from 2.9.1
2. Updated versions
3. add px-role with secrets
4. Added artifacts

px-role:
```
 k -n portworx get role px-role -oyaml 
apiVersion: rbac.authorization.k8s.io/v1
kind: Role
metadata:
  annotations:
    meta.helm.sh/release-name: v2-1658218410
    meta.helm.sh/release-namespace: kube-system
  creationTimestamp: "2022-07-19T08:13:31Z"
  labels:
    app.kubernetes.io/managed-by: Helm
  managedFields:
  - apiVersion: rbac.authorization.k8s.io/v1
    fieldsType: FieldsV1
    fieldsV1:
      f:metadata:
        f:annotations:
          .: {}
          f:meta.helm.sh/release-name: {}
          f:meta.helm.sh/release-namespace: {}
        f:labels:
          .: {}
          f:app.kubernetes.io/managed-by: {}
      f:rules: {}
    manager: helm
    operation: Update
    time: "2022-07-19T08:13:31Z"
  name: px-role
  namespace: portworx
  resourceVersion: "3464"
  uid: 3cebceda-6ff5-44eb-9f83-99b2659d64b1
rules:
- apiGroups:
  - ""
  resources:
  - secrets
  verbs:
  - get
  - list
  - create
  - update
  - patch
  - delete
```
